hw1/02-palindrome.js

```diff
@@ -1,3 +1,40 @@
-const elem = document.querySelector('input');
+const integerInputElement = document.getElementById("integer_input");
+const resultElement = document.getElementById("result");
 
-elem.addEventListener('input', handleInput);
+integerInputElement.addEventListener("input", handleInput);
+
+function handleInput() {
+  let value = integerInputElement.value;
+  if (value < 0) {
+    resultElement.innerHTML = `<p style="color:red">Enter a positive number</p>`;
+  } else if (value.length === 0) {
+    resultElement.innerHTML = "";
+  } else if (isAlphabetsPresent(value)) {
+    resultElement.innerHTML = `<p style="color:red">Please enter a number.</p>`;
+  } else if (isPalindrome(value)) {
+    resultElement.innerHTML = `<p style="color:green">Yes. This is a palindrome!</p>`;
+  } else {
+    resultElement.innerHTML = `<p style="color:red">No. Try again.</p>`;
+  }
+}
+
+function isAlphabetsPresent(value) {
+  var regExp = /[a-zA-Z]/g;
+  return regExp.test(value);
+}
+
+function isPalindrome(value) {
```

Comment:
I don't believe that these function headers are in line with the AirBnB style guide so you might want to reference that again.